packages/core/src/lib/markdown-it/markdown-it-icons.js

```diff
@@ -10,11 +10,23 @@ module.exports = require('markdown-it-regexp')(
         if (iconFontType === 'glyphicon') {
             return `<span aria-hidden="true" class="glyphicon glyphicon-${iconFontName}"></span>`;
         } else if (iconFontType === 'octicon') {
-            return iconClass ? octicons[iconFontName].toSVG({"class": iconClass})
-                             : octicons[iconFontName].toSVG();
+            let icon = iconClass
+                ? octicons[iconFontName].toSVG({"class": iconClass})
+                : octicons[iconFontName].toSVG();
+            // ensure octicons are valid
+            if (!octicons.hasOwnProperty(iconFontName)) {
```

Comment:
should this be above `octicons[iconFontName]`? 😮

Reply:
No, it shouldn't ... so sorry about that 😓

Reply:
corrected!